lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -21,10 +21,21 @@ class BaseI18nTestCase(TestCase):
     """
     Base utilities for i18n test classes to derive from
     """
+    def __init__(self, *args, **kwargs):
```

Comment:
It was something I inherited with the original tests.  I have updated and removed the init method again.  Everything is not in setup that needs to be.